public/assets/js/scripts.js

```diff
@@ -285,4 +285,27 @@ $(document).ready(function () {
     $(this).html() == '<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>' ? $(this).html('<span><img src="./assets/img/icons/view-map-pin.svg" alt="view map"> &nbsp; View Map</span>') : $(this).html('<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>');
   });
 
+  $('.search-calculate-button').click(function(){
+    let periodic = $('.periodic-investment').val();
+    let initial = $('.initial-investment').val();
+    let maxValue = removeCommasAndMakeNumber($('.investment-value-input').data('max-amount'));
+    let frequency = $("input[name='investment-frequency']:checked").val();
+  
+    $.ajax({
```

Comment:
Nice thought with this; however, since we are not getting anything from the database, there is no need to process this in PHP when you can also write the same function in Javascript. The disadvantage of making an ajax call (when it could be solved without one) is that it will take a long time to calculate for people with very slow internet connections (since an additional request is being made to the server). 

Moreover, it is a better system design to use Javascript for the calculation (client-side processing is faster than server-side), it will load faster, less prone to network errors and still output the same result.